src/status_im/wallet/accounts/core.cljs

```diff
@@ -21,15 +21,17 @@
 
 (re-frame/reg-fx
  ::generate-account
- (fn [{:keys [address hashed-password path-num]}]
-   (status/multiaccount-load-account
-    address
-    hashed-password
-    (fn [value]
-      (let [{:keys [id error]} (types/json->clj value)]
-        (if error
-          (re-frame/dispatch [::generate-new-account-error])
-          (let [path (str constants/path-root "/" path-num)]
+ (fn [{:keys [derivation-info
```

Comment:
just wondering why keys in a column and not in a row?

Reply:
i like columns :) but i can put them on a row, np